benches/normalize.rs

```diff
@@ -19,13 +20,21 @@ where
         libc::dlopen as Addr,
         libc::fopen as Addr,
         normalize_process_impl::<M> as Addr,
-        Normalizer::normalize_user_addrs_sorted as Addr,
+        Normalizer::normalize_user_addrs as Addr,
     ];
     let () = addrs.sort();
+    let opts = NormalizeOpts {
+        sorted_addrs: true,
+        ..Default::default()
+    };
 
     let () = b.iter(|| {
         let normalized = normalizer
-            .normalize_user_addrs_sorted(black_box(0.into()), black_box(addrs.as_slice()))
+            .normalize_user_addrs_opts(
+                black_box(0.into()),
```

Comment:
curious, why using black_box() here?

Reply:
We don't want the optimizer to inline everything and potentially specialize the code for the arguments in question -- which are available at compile/link time -- as that could perturb the reported numbers in a positive way not necessarily representative of real world usage (where data may only be known at runtime).